src/lib/structures/ModerationCommand.ts

```diff
@@ -145,3 +120,38 @@ export abstract class ModerationCommand<T = unknown> extends SkyraCommand {
 	public abstract posthandle(message: KlasaMessage, targets: User[], reason: string | null, prehandled: T): unknown;
 
 }
+
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export async function checkModeratable<T>(message: KlasaMessage, target: User, requiredMember: boolean, client: KlasaClient, _prehandled?: T) {
```

Comment:
You should retrieve client from `message`. Also, I'd make this an static method on ModerationCommand, so this is namespaced.